compiler/src/dotty/tools/dotc/ast/Trees.scala

```diff
@@ -677,6 +677,29 @@ object Trees {
     override def isType = expansion.isType
   }
 
+  /** A tree representing a quote `'{ expr }
```

Comment:
For both Quote and Splice, it would be nice to document in which compiler phases they exist, and given a quick overview of how they are transformed.